bulk_query_gen/influxdb/influx_iot_stand_alone_filter.go

```diff
@@ -0,0 +1,31 @@
+package influxdb
```

Comment:
This file was renamed from `influx_iot_light_level.go`, with the `LightLevel` replaced with `StandAloneFilter`